src/js/view/captionsrenderer.js

```diff
@@ -193,6 +193,16 @@ define([
             this.populate(_model.get('captionsTrack'));
         };
 
+        this.clear = function () {
+            while(_display.firstChild) {
+                _display.removeChild(_display.firstChild);
+            }
```

Comment:
Use `utils.empty`. It's already required and used in this module. It's part of dom.js:
```
    dom.empty = function(element) {
        if (!element) {
            return;
        }
        while (element.childElementCount > 0) {
            element.removeChild(element.children[0]);
        }
    };
```